fwutil/main.py

```diff
@@ -348,6 +349,13 @@ def show():
 def updates(ctx, image):
     """Show available updates"""
     try:
+        # Barefoot
+        if(str(device_info.get_platform()) == "x86_64-accton_as9516bf_32d-r0"   or
```

Comment:
the expression can be simplified 
```
platforms = {
  "x86_64-accton_as9516bf_32d-r0",
  "x86_64-accton_as9516_32d-r0",
  "x86_64-accton_wedge100bf_32x-r0",
  "x86_64-accton_wedge100bf_65x-r0",
}
if device_info.get_platform() in platforms :
    ...
```

Reply:
What if a new barefoot platform will be added?
Please use `get_platform_info` and check by `asic_type` filed. https://github.com/msosyak/sonic-buildimage/blob/2e7e7814cc71c353bef179c7f29bf00fcbac8714/src/sonic-py-common/sonic_py_common/device_info.py#L327

Reply:
done